CMakeLists.txt

```
@@ -436,7 +436,10 @@ if(SFML_OS_WINDOWS)
 
     if(NOT SFML_USE_SYSTEM_DEPS)
         # install the binaries of SFML dependencies
-        if(ARCH_32BITS)
+        if(ARCH_ARM64)
+            install(DIRECTORY extlibs/bin/ARM64/ DESTINATION ${CMAKE_INSTALL_BINDIR})
+            install(DIRECTORY extlibs/libs-msvc-universal/ARM64/ DESTINATION ${CMAKE_INSTALL_LIBDIR})
```

```suggestion
            install(DIRECTORY extlibs/libs-msvc-universal/Arm64/ DESTINATION ${CMAKE_INSTALL_LIBDIR})
```
https://www.arm.com/company/policies/trademarks/guidelines-trademarks#:~:text=Never%20alter%20Arm's%20trademarks%3A,)%2C%20or%20using%20improper%20capitalization.

Arm themselves don't capitalize all three letters. I think it's correct to keep using all caps for CMake variables but I think for directories we can use the more proper "Arm" style.

---

It's tricky. If you look around you'll find a lot of different handling of this.

I searched a bit earlier today and came to the conclusion that ARM64 is a good fit. Windows documentation and IDE mostly refer to it as ARM64, so does CMake, etc.

---

Fine by me if we stick with the all-caps directory name.